internal/storage/db-storage.go

```diff
@@ -0,0 +1,323 @@
+package storage
+
+import (
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/anatoly32322/metriccollector/internal/logger"
+	"github.com/anatoly32322/metriccollector/internal/retry"
+	"strconv"
+	"sync"
+)
+
+var (
```

Comment:
Стоит использовать `const` если мы не намереваемся изменять эти строки.
```suggestion
const (
```